qa/TL3_JAX_multiprocess/test_jax.sh

```diff
@@ -6,4 +6,22 @@ function CLEAN_AND_EXIT {
 
 python -c "import jax; print(jax.devices()); assert jax.device_count() > 0"
 
+echo "Test one GPU per process"
+CUDA_VISIBLE_DEVICES="1" timeout -k 60s 60s python jax_client.py --id 1 --size 8 &
+CUDA_VISIBLE_DEVICES="2" timeout -k 60s 60s python jax_client.py --id 2 --size 8 &
+CUDA_VISIBLE_DEVICES="3" timeout -k 60s 60s python jax_client.py --id 3 --size 8 &
+CUDA_VISIBLE_DEVICES="4" timeout -k 60s 60s python jax_client.py --id 4 --size 8 &
+CUDA_VISIBLE_DEVICES="5" timeout -k 60s 60s python jax_client.py --id 5 --size 8 &
+CUDA_VISIBLE_DEVICES="6" timeout -k 60s 60s python jax_client.py --id 6 --size 8 &
+CUDA_VISIBLE_DEVICES="7" timeout -k 60s 60s python jax_client.py --id 7 --size 8 &
+CUDA_VISIBLE_DEVICES="0" timeout -k 60s 60s python jax_server.py --size 8 &
+
+wait $(jobs -p)
+
+echo "Test multiple GPUs per process"
+CUDA_VISIBLE_DEVICES="4,5,6,7" timeout -k 60s 60s python jax_client.py --id 1 --size 2 &
+CUDA_VISIBLE_DEVICES="0,1,2,3" timeout -k 60s 60s python jax_server.py --size 2
+
+wait $(jobs -p)
```

Comment:
This is probably my fault, but I think we will check for the status of the wait instead of the jax_server invokation. Should we check for correct execution of all the clients and servers by checking their exit codes?